### DISCUSSION TITLE
Accounting KPI Dashboard

### DISCUSSION THREAD

**Discussion by etobella** [OCA/odoo-community.org#86] [Ideas]
With 16.0, we are able to create and edit Spreadsheets. On some previous PRs, we were unable to use Odoo original spreadsheet, however, we are able to create our own one. Before doing this, we need to define all possible KPIS and how to compute them :smile: 

For example, using account groups and balances:

- Gross Profit: income - expense_direct_cost
- Net Profit: income +income_other- expense_direct_cost - expense - expense_depreciation
- Gross profit margin: Gross Profit / income
- Net Profit margin: Net profit / (income + incomet_other)
- Current assets: asset_receivable + asset_cash + asset_current + asset_prepayments 
- Current Liabilities: liability_payable + liabiloity_credit_card + liability_current
- Working capital: Current assets - Current Liabilities

More ideas?

I computed most of them on a small spreadsheet, so, not too hard to create the module :smile:

**Comment by pedrobaeza** [comment]
Yes, similar to the thread I opened: https://odoo-community.org/groups/contributors-15/contributors-257148?mode=thread&date_begin=&date_end=

so for me, go ahead!
